definitions/npm/effector-react_v22.x.x/flow_v0.83.x-/effector-react_v22.x.x.js

```diff
@@ -0,0 +1,119 @@
+declare module 'effector-react' {
+  //@flow strict
+
+  import type {
+    ComponentType,
+    Component,
+    ElementConfig,
+    Node,
+    Context as ContextType,
+  } from 'react'
+  import type {Store, Event} from 'effector'
+
+  export type StoreConsumer<State> = ComponentType<{|
```

Comment:
All these exports are missing the `declare` keyword which is necessary inside module declarations.